docs/settings.rst

```diff
@@ -1067,6 +1067,11 @@ as follows::
   )
 
 
+If you want a pattern to apply to the last page in the list, use ``-1``
+as ``minimum page`` value like this::
```

Comment:
nit: Could this be worded as _"use `-1` as the `minimum_page` value:"_ instead?

Reply:
Sure! I have updated this line and the PR title and description. Please have a look.